### PR TITLE
Chore: Report betterer stats in ci-frontend-metrics

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

Parses the .betterer.results files and reports them to graphite in ci-frontend-metrics

**Which issue(s) this PR fixes**:

Relates to #42385

**Special notes for your reviewer**:

And let me tell you, bash is a _chore_.

### CODE REVIEW

File: scripts/ci-frontend-metrics.sh [hunk 1]
```diff
@@ -38,7 +40,17 @@ echo -e "High vulnerabilities: $HIGH_VULNERABILITIES"
 echo -e "Critical vulnerabilities: $CRITICAL_VULNERABILITIES"
 echo -e "Number of enzyme tests: $ENZYME_TEST_COUNT"
 
+BETTERER_STATS=""
+
+while read line || [[ -n $line ]]
```
Comment (rlankfo) [inline scripts/ci-frontend-metrics.sh:45]: What value does the `|| [[ -n $line ]]` provide?  If `read line` isn't successful, `line` won't be set.

File: scripts/ci-frontend-metrics.sh [hunk 2]
```diff
@@ -38,7 +40,17 @@ echo -e "High vulnerabilities: $HIGH_VULNERABILITIES"
...
+do
+  read -r name value <<< "$line"
```
Comment (rlankfo) [inline scripts/ci-frontend-metrics.sh:47]:
```suggestion
while read -r name value
do
```

File: scripts/ci-frontend-metrics.sh (hunk 1 above)
Comment (joshhunt) [reply scripts/ci-frontend-metrics.sh]: Fair! Thanks for the feedback!

File: scripts/cli/reportBettererStats.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { betterer } from '@betterer/betterer';
+import { camelCase } from 'change-case';
```
Comment (jackw) [inline scripts/cli/reportBettererStats.ts:2]: Is there any reason we aren't using lodash `camelCase` here to save on adding more dependencies to the project?

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: scripts/cli/reportBettererStats.ts (hunk 3 above)
Comment (joshhunt) [reply scripts/cli/reportBettererStats.ts:2]: Good shout!!!!

**Pull request merged**